tests/org.jboss.tools.drools.ui.bot.test/src/org/jboss/tools/drools/ui/bot/test/kienavigator/ServerConnectionTest.java

```diff
@@ -1,51 +0,0 @@
-package org.jboss.tools.drools.ui.bot.test.kienavigator;
-
-import java.util.List;
-
-import org.eclipse.reddeer.junit.requirement.inject.InjectRequirement;
-import org.eclipse.reddeer.junit.runner.RedDeerSuite;
-import org.eclipse.reddeer.requirements.server.ServerRequirementState;
-import org.jboss.tools.drools.reddeer.kienavigator.item.OrgUnitItem;
-import org.jboss.tools.drools.reddeer.kienavigator.properties.ServerProperties;
-import org.jboss.tools.drools.reddeer.view.KieNavigatorView;
-import org.jboss.tools.runtime.reddeer.requirement.ServerRequirement;
-import org.jboss.tools.runtime.reddeer.requirement.ServerRequirement.Server;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-@Server(state = ServerRequirementState.RUNNING)
-@RunWith(RedDeerSuite.class)
-public class ServerConnectionTest extends KieNavigatorTestParent {
```

Comment:
@tomasdavidorg Why did you remove this test?

Reply:
@apodhrad It tests that error appears when incorrect values are set. It does not make sense to keep test like this one.